azureadps-1.0/MSOnline/Set-MsolDirSyncFeature.md

```diff
@@ -23,13 +23,12 @@ The **Set-MsolDirSyncFeature** cmdlet sets identity synchronization features for
 
 Synchronization features that can be used with this cmdlet include the following:
 
-- EnableSoftMatchOnUpn. Soft Match is the process used to link an object being synced from on-premises for the first time with one that already exists in the cloud. When this feature is enabled Soft Match will first be attempted using the standard logic, based on primary SMTP address. If a match is not found based on primary SMTP, then a match will be attempted based on UserPrincipalName. Once this feature is enabled it cannot be disabled.
-- PasswordSync
-- SynchronizeUpnForManagedUsers. Allows for the synchronization of UserPrincipalName updates from on-premises for managed (non-federated) users that have been assigned a license. These updates will be blocked if this feature is not enabled. Once this feature is enabled it cannot be disabled.
-- BlockSoftMatch. When this feature is enabled it will block the Soft Match feature. Customers are encouraged to enable this feature and keep it at enabled until Soft Matching is required again for their tenancy. This flag should be enabled again after any soft matching has completed and is no longer needed.
-- BlockCloudObjectTakeoverThroughHardMatch. When this feature is enabled and an object is synced for which an object with a matching source anchor already exists in AAD and that object in AAD does not have DirSyncEnabled set to "true", the default behaviour would be to hard match the cloud object with the on premises object and set the DirSyncEnabled flag of the Cloud object to "true". <br>
-When enabling this feature the cloud object is no longer matched and the DirSyncEnabled flag is not set to "True". Instead, an error is thrown: Error Code: InvalidHardMatch, Error Message: Another cloud created object with the same source anchor already exists in Azure Active Directory. 
-
+- **EnableSoftMatchOnUpn**: Soft Match is the process used to link an object being synced from on-premises for the first time with one that already exists in the cloud. When this feature is enabled Soft Match will first be attempted using the standard logic, based on primary SMTP address. If a match is not found based on primary SMTP, then a match will be attempted based on UserPrincipalName. Once this feature is enabled it cannot be disabled.
+- **PasswordSync**
+- **SynchronizeUpnForManagedUsers**: Allows for the synchronization of UserPrincipalName updates from on-premises for managed (non-federated) users that have been assigned a license. These updates will be blocked if this feature is not enabled. Once this feature is enabled it cannot be disabled.
+- **BlockSoftMatch**: When this feature is enabled it will block the Soft Match feature. Customers are encouraged to enable this feature and keep it at enabled until Soft Matching is required again for their tenancy. This flag should be enabled again after any soft matching has completed and is no longer needed.
```

Comment:
```suggestion
- **BlockSoftMatch**: When this feature is enabled it will block the soft match feature. Customers are encouraged to enable this feature and keep it at enabled until soft matching is required again for their tenancy. This flag should be enabled again after any soft matching has completed and is no longer needed.
```